web/grunt/karma-config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function (grunt) {
+    'use strict';
+
+    const karmaTasksConfig = {
+        browsers: ['Firefox'],
+        frameworks: ['jasmine', 'browserify'],
+        preprocessors: {
+            '<%= config.tests %>': ['browserify'],
+            '<%= config.compiledScriptsDir %>/**/*.js': ['browserify']
+        },
+        browserify: {
+            debug: true,
+            paths: '<%= config.scriptsDir %>'
+        }
+    };
+
+    const configure = function () {
+        grunt.config.merge({
+            karma: {
+                options: {
+                    files: ['<%= config.compiledScriptsDir %>/**/*.js', '<%= config.tests %>']
+                },
+                dev: karmaTasksConfig,
+                ci: Object.assign(
+                    {
+                        options: {
+                            singleRun: true
+                        }
+                    },
+                    karmaTasksConfig
+                )
+            }
+        });
+    };
+
+    configure();
```

Comment:
Here I won't agree. For me it's a bit cleaner to define functions and then call it, than have some code floating around. If you wanted to add something more, before the configure step for example, it would be much nicer to add a function and then call it lile:
```
prepare();
configure();
```
than just add some more floating code. I think it helps with organisation.